### PR TITLE
New Rule (Link): Office document loads remote HTML template

### PR DESCRIPTION
**Pull request by aidenmitchell**

### CODE REVIEW

**Comment by morriscode** [comment]
heads up, not ignoring this, it's just really expensive on backtesting, so I haven't had much luck validating tp/fp ratios.

**Comment by morriscode** [comment]
😭  On a 3 day retro, returned 9 FP's and no TP's. 
Lots of docx's with meeting agendas or forms. Might need to refactor our approach here

**Comment by aidenmitchell** [comment]
How do we feel about adding the FTS snippet?

**Comment by morriscode** [comment]
> How do we feel about adding the FTS snippet?

We can give it a go!

File: detection-rules/link_office_remote_html_template.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+source: |
+  type.inbound 
+  and any(body.links, strings.ilike(.display_url.url, "*doc*", "*docm*", "*docx*", "*dot*", "*dotm*", "*pptm*", "*ppsm*", "*xlm*", "*xls*", "*xlsb*", "*xlsm*", "*xlt*", "*xltm*")
+        and any(beta.linkanalysis(.).files_downloaded, 
+        (
+        (
+            // office files
+            .file_extension in~ ("doc", "docm", "docx", "dot", "dotm", "pptm", "ppsm", "xlm", "xls", "xlsb", "xlsm", "xlt", "xltm")
+        )
+        and any(file.oletools(.).relationships, regex.icontains(.target, "html?!"))
```
Comment (morriscode) [inline detection-rules/link_office_remote_html_template.yml:34]:
```suggestion
        and any(file.oletools(.).relationships, regex.icontains(.target, "html?"))
```

Any idea what the ! point was for?

File: detection-rules/link_office_remote_html_template.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+            // rtf files
+            or .file_extension in~ ("rtf")
+        )
+
+        and (
+          any(file.explode(.),
+              (
+                  // office files
+                  .flavors.mime == "text/xml" and
+                  any(.scan.strings.strings, regex.icontains(., "oleObject.*html?!"))
```
Comment (morriscode) [inline detection-rules/link_office_remote_html_template.yml:50]:
```suggestion
                  any(.scan.strings.strings, regex.icontains(., 'oleObject\.*html?'))
```

File: detection-rules/link_office_remote_html_template.yml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+              ) or
+              (
+                  // rtf files
+                  .flavors.mime == "text/rtf" and
+                  any(.scan.strings.strings, regex.icontains(., "objclass.*htm.?"))
```
Comment (morriscode) [inline detection-rules/link_office_remote_html_template.yml:55]:
```suggestion
                  any(.scan.strings.strings, regex.icontains(., 'objclass\.*html?'))
```

**Pull request closed without merging**